internal/normalizer/normalizer.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Mainflux
```

Comment:
I think that the normalizer shouldn't be in the `internal` package. AFAIK, everything that's in the `internal` cannot be used from the outside and as far as I understood, we want this lib to be usable from the outside too. At least think about creating an interface for the normalizer. Maybe this interface should have:
```
Normalize(msg mainflux.RawMessage) ([]interface{}, error)
```

You'll later have to cast this `interface{}` but that's part of the language (if it had generics maybe we could have done something about it). This way you'll be able to create an interface for any other message type.